inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.h

```diff
@@ -49,6 +49,7 @@ class MKLDNNPadNode : public MKLDNNNode {
         InferenceEngine::SizeVector srcStrides;
         InferenceEngine::SizeVector dstStrides;
         InferenceEngine::SizeVector srcDimsForReflectOrSymmetric;
+        size_t nThreads = 0lu;
```

Comment:
Does it make sense to use size_t? The only function this argument is sent to is `parallel_nt`, which takes `int`.

Reply:
Changed on `int`, thanks